js/main.js

```diff
@@ -26,7 +26,8 @@ jQuery(function($) {
 				var $this = $(this);
 
 				// add a tab head link for each tab
-				tabsHead.append('<a href="#">' + $this.find('.gsection_title').html() + '</a>');
+				// add the id as well so gf conditional sections still works
+				tabsHead.append('<a id="' + $this[0].id + '" href="#">' + $this.find('.gsection_title').html() + '</a>');
```

Comment:
Howdy @recyclingrules, this link should allow you to download the modified code:

https://github.com/gubbigubbi/gravity-forms-section-tabs/archive/77d323d649df0594352e9ca698d47d08601a611b.zip

In the meantime, I'd be happy to assist with landing this one if @gubbigubbi would need some help. I'll probably do that anyway if we don't get any movement soon.